src/core/components/time/time.component.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+/**
+ * @ngdoc component
+ * @name udb.search.directive:udbSearchBar
+ * @description
+ * # udbQuerySearchBar
+ */
+angular
+  .module('udb.core')
+  .component('udbTime', {
```

Comment:
Van deze component maak je beter een simpele directive. Het enige wat je dan moet doen is inhaken op de [model controller](https://docs.angularjs.org/api/ng/type/ngModel.NgModelController) om je data te formatten. Er is geen speciale markup. Je kan dan ook de gewenste callback methode meespelen vanuit de context.